### PR TITLE
🌐 [i18n-KO] Translated `serialization.mdx` to Korean

### PR DESCRIPTION
**Pull request by wonhyeongseo**
# What does this PR do?

Translated the `serialization.mdx` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Team PseudoLab, may you please review this PR? @0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

@sgugger, @ArthurZucker, @eunseojo May you please review this PR?

### CODE REVIEW

**Comment by wonhyeongseo** [comment]
Team PseudoLab, may you please review this PR? @0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

File: docs/source/ko/serialization.mdx [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+>>> validate_model_outputs(
+...     onnx_config, tokenizer, base_model, onnx_path, onnx_outputs, onnx_config.atol_for_validation
+... )
+```
+
+이 함수는 [`~transformers.onnx.OnnxConfig.generate_dummy_inputs`] 메서드로 기존 및 내보낸 모델의 입력을 생성하며, 검증에 사용될 오차 범위는 구성에서 정의할 수 있습니다. 일반적으로는 1e-6에서 1e-4 범위 내에서 합의하지만, 1e-3보다 작다면 문제 없을 가능성이 높습니다.
+
+## 🤗 Transformers에 새 구성 추가하기[[contributing-a-new-configuration-to-transformers]]
+
+이미 만들어진 구성의 숫자를 늘리려고 노력하고 있으며, 커뮤니티의 기여를 환영합니다! 라이브러리에 당신만의 구성을 추가하려면 다음 단계를 기억해주세요:
```
Comment (sim-so) [inline docs/source/ko/serialization.mdx:445]:
```suggestion
미리 제작된 구성의 숫자를 늘리려고 노력하고 있으며, 커뮤니티의 기여를 환영합니다! 라이브러리에 당신만의 구성을 추가하려면 다음 단계를 기억해주세요:
```
`ready-made`를 앞에서 사용된 번역어와 맞추었습니다.

File: docs/source/ko/serialization.mdx [hunk 2]
```diff
@@ -0,0 +1,451 @@
...
+
+>>> config = AutoConfig.from_pretrained("distilbert-base-uncased")
+>>> onnx_config_for_seq_clf = DistilBertOnnxConfig(config, task="sequence-classification")
+>>> print(onnx_config_for_seq_clf.outputs)
+OrderedDict([('logits', {0: 'batch'})])
+```
+
+<Tip>
+
+[`~onnx.config.OnnxConfig`]나 다른 구성 클래스에 연결된 모든 기본 속성 및 메서드는 필요에 따라 모두 재정의할 수 있습니다. 고급 예제로 [`BartOnnxConfig`]를 확인하세요.
```
Comment (sim-so) [inline docs/source/ko/serialization.mdx:393]:
```suggestion
[`~onnx.config.OnnxConfig`]나 다른 구성 클래스에 연결된 모든 기본 속성 및 메소드는 필요에 따라 모두 재정의할 수 있습니다. 고급 예제로 [`BartOnnxConfig`]를 확인하세요.
```

**Review by sim-so — CHANGES_REQUESTED** [review]
원형님의 번역 덕분에 많은 내용을 새로 알게 되었어요! 
수식어절이 긴 문장을 나누어 옮겨주신 점이 좋아서 저도 다음 번역에 참고하려고 해요 🙌 

`:` 뒤에 예시가 이어지는 문장과 몇 가지 표현에 대해 제안을 드립니다.
다른 의견이 있는 경우 편히 제안해주시면 좋겠습니다!

File: docs/source/ko/serialization.mdx [hunk 3]
```diff
@@ -0,0 +1,451 @@
...
+```
+
+이 함수는 [`~transformers.onnx.OnnxConfig.generate_dummy_inputs`] 메서드로 기존 및 내보낸 모델의 입력을 생성하며, 검증에 사용될 오차 범위는 구성에서 정의할 수 있습니다. 일반적으로는 1e-6에서 1e-4 범위 내에서 합의하지만, 1e-3보다 작다면 문제 없을 가능성이 높습니다.
+
+## 🤗 Transformers에 새 구성 추가하기[[contributing-a-new-configuration-to-transformers]]
+
+이미 만들어진 구성의 숫자를 늘리려고 노력하고 있으며, 커뮤니티의 기여를 환영합니다! 라이브러리에 당신만의 구성을 추가하려면 다음 단계를 기억해주세요:
+
+* `configuration_<model_name>.py` 파일에 ONNX 구성을 구현하세요.
+* [`~onnx.features.FeatureManager`]에 모델 아키텍처 및 해당 기능을 포함시키세요.
```
Comment (0525hhgus) [inline docs/source/ko/serialization.mdx:448]:
```suggestion
* [`~onnx.features.FeatureManager`]에 모델 아키텍처 및 해당 기능을 포함하세요.
```
전후 문장과 유사하게 능동으로 제안드립니다!

**Review by 0525hhgus — CHANGES_REQUESTED** [review]
번역 감사합니다! 문서 내 코드와 함께 표현을 고려하신 점이 느껴집니다 👍
저는 마지막 문단에 짧은 리뷰 덧붙였습니다 😄

File: docs/source/ko/serialization.mdx [hunk 4]
```diff
@@ -0,0 +1,451 @@
...
+
+## 다른 모델 작업에 대한 기능 선택[[selecting-features-for-different-model-tasks]]
+
+<Tip>
+
+모델을 내보내는 권장 방법은 이제 `optimum.exporters.onnx`를 사용하는 것입니다. 작업을 선택하는 방법을 알아보려면 [🤗 Optimum 문서](https://huggingface.co/docs/optimum/main/en/exporters/onnx/usage_guides/export_a_model#selecting-a-task)를 확인하세요.
+
+</Tip>
+
+각 미리 제작된 구성에는 다른 유형의 작업에 대한 모델을 내보내도록 하는 일련의 _기능_이 포함되어 있습니다. 아래 표에 나와 있는대로 각 기능은 다른 `AutoClass`와 연관되어 있습니다.
```
[the remark this answers is not in the document]
Comment (wonhyeongseo) [reply docs/source/ko/serialization.mdx:246]:
```suggestion
다른 유형의 태스크에 맞춰서 모델을 내보낼 수 있도록 미리 제작된 구성마다 일련의 _기능_이 포함되어 있습니다. 아래 표에 나와 있는대로 각 기능은 다른 `AutoClass`와 연관되어 있습니다.
```

File: docs/source/ko/serialization.mdx [hunk 5]
```diff
@@ -0,0 +1,451 @@
...
+```
+
+다음과 같이 모델에 연결된 출력을 볼 수도 있습니다:
+
+```python
+>>> print(onnx_config.outputs)
+OrderedDict([("last_hidden_state", {0: "batch", 1: "sequence"})])
+```
+
+출력 속성이 입력과 동일한 구조임을 유의하세요. 각 출력은 이름과 차원이 `OrderedDict`의 키-값으로 저장되어 있습니다. 출력 구조는 구성을 초기화할 때 선택한 기능과 관련이 있습니다. 기본적으로 ONNX 구성은 `AutoModel` 클래스로 가져온 모델을 내보낼 때 쓰이는 `default` 기능으로 초기화됩니다. 다른 태스크를 위해 모델을 내보내려면 ONNX 구성을 초기화할 때 `task` 인수에 다른 기능을 넣으면 됩니다. 예를 들어, 앞에 시퀀스 분류를 하도록 DistilBERT를 내보내려면:
```
[the remark this answers is not in the document]
Comment (wonhyeongseo) [reply docs/source/ko/serialization.mdx:380]:
```suggestion
출력 속성이 입력과 동일한 구조임을 유의하세요. 각 출력은 이름과 차원이 `OrderedDict`의 키-값으로 저장되어 있습니다. 출력 구조는 구성을 초기화할 때 선택한 기능과 관련이 있습니다. 기본적으로 ONNX 구성은 `AutoModel` 클래스로 가져온 모델을 내보낼 때 쓰이는 `default` 기능으로 초기화됩니다. 다른 태스크를 위해 모델을 내보내려면 ONNX 구성을 초기화할 때 `task` 인수에 다른 기능을 넣으면 됩니다. 예를 들어, 시퀀스 분류 단계를 덧붙인 DistilBERT를 내보내려면, 이렇게 해볼 수 있습니다:
```

File: docs/source/ko/serialization.mdx [hunk 6]
```diff
@@ -0,0 +1,451 @@
...
+```python
+>>> import onnx
+
+>>> onnx_model = onnx.load("model.onnx")
+>>> onnx.checker.check_model(onnx_model)
+```
+
+<Tip>
+
+모델 크기가 2GB보다 큰 경우 내보내기 중에 여러 추가 파일들이 생성되는 것을 볼 수 있습니다. 사실 ONNX는 모델을 저장하기 위해 [Protocol Buffers](https://developers.google.com/protocol-buffers/)를 사용하는데, 버퍼는 2GB의 크기 제한이 있기 때문에 _예상대로_의 작동입니다. 외부 데이터를 사용하여 모델을 가져오는 방법은 [ONNX 문서](https://github.com/onnx/onnx/blob/master/docs/ExternalData.md)를 참조하세요.
```
[the remark this answers is not in the document]
Comment (wonhyeongseo) [reply docs/source/ko/serialization.mdx:425]:
```suggestion
모델 크기가 2GB보다 큰 경우 내보내는 중에 여러 추가 파일들이 생성되는 것을 볼 수 있습니다. 사실 ONNX는 모델을 저장하기 위해 [Protocol Buffers](https://developers.google.com/protocol-buffers/)를 사용하는데, 버퍼는 2GB의 크기 제한이 있기 때문에 _자연스러운_ 일입니다. 외부 데이터를 사용하여 모델을 가져오는 방법은 [ONNX 문서](https://github.com/onnx/onnx/blob/master/docs/ExternalData.md)를 참조하세요.
```

**Review by wonhyeongseo — COMMENTED** [review]
Thank you @sim-so and @0525hhgus for your wonderful reviews. Good night 🌙

**Comment by wonhyeongseo** [comment]
@sgugger, @ArthurZucker, @eunseojo May you please review this PR?

**Review by sgugger — APPROVED** [review]
Thanks!
